spec/controllers/albums_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+require 'simplecov'
+
+RSpec.describe AlbumsController, type: :controller do
+  it_behaves_like "a controller" do
+    let(:medium) do
+      Album.create(title: "dumb")
+    end
+    let(:params) do
+      {
+          album: {
+          title: "a title",
+          artist: "a music artist",
+          description: "a description"
+        }
+      }
```

Comment:
Try using a hash rocket here instead if you're having trouble getting the strong params to work in your shared rspec examples. I remember having a similar problem with my code and using a hash rocket helped for some reason. Although Ruby is moving away from the hash rocket, there are still some weird instances where you still have to use them. Maybe it might work for you?

```ruby
album => {
  title: "a title",
  artist: "a music artist",
  description: "a description"
}
```